### PR TITLE
Add Skillsoft to `ADOPTERS.md`

### PR DESCRIPTION
**Pull request by brandon-powers**
### Description of your changes

👋 

This PR adds **[Skillsoft](https://www.skillsoft.com/)** to `ADOPTERS.md`, as we've recently been using Crossplane in production to provision the underlying AWS infrastructure for core Kafka services. 

Currently, we're using the S3, Athena, and Glue providers, though we'll likely increase usage later this year.  

I have: 

- [X] Read and followed Crossplane's [contribution process].
- [ ] ~Run `make reviewable` to ensure this PR is ready for review.~
- [ ] ~Added or updated unit tests.~
- [ ] ~Added or updated e2e tests.~
- [ ] ~Linked a PR or a [docs tracking issue] to [document this change].~
- [ ] ~Added `backport release-x.y` labels to auto-backport this PR.~

Need help with this checklist? See the [cheat sheet].

[contribution process]: https://github.com/crossplane/crossplane/tree/master/contributing
[docs tracking issue]: https://github.com/crossplane/docs/issues/new
[document this change]: https://docs.crossplane.io/contribute/contribute
[cheat sheet]: https://github.com/crossplane/crossplane/tree/master/contributing#checklist-cheat-sheet

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -27,62 +27,63 @@ updating this list.
...
+| ------------ | ------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (jbw976) [inline ADOPTERS.md:31]: All the other lines of this file were updated - did this come from an automatic formatting tool you have installed in your environment maybe? wanting to make sure nothing else was edited about each of the lines showing up in the diff 🤔

File: ADOPTERS.md [hunk 2]
```diff
@@ -27,62 +27,63 @@ updating this list.
...
+| [Skillsoft](https://www.skillsoft.com/) | [@brandon-powers](https://github.com/brandon-powers) | At Skillsoft, Crossplane provisions & manages the AWS infrastructure (S3, Athena, and Glue) required by core Apache Kafka services we maintain to power our online learning platform, [Percipio](https://www.skillsoft.com/meet-skillsoft-percipio).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (jbw976) [inline ADOPTERS.md:89]: > we've recently been using Crossplane in production

This is awesome! could you update this description with this same production usage detail, so folks will see that when they look at this list later on 😉 

something like:

```
to power our online learning platform, [Percipio](https://www.skillsoft.com/meet-skillsoft-percipio), in production environments.
```

**Review by jbw976 — COMMENTED** [review]
Awesome @brandon-powers, thanks for taking the time to share your crossplane story with the community! a couple questions for you in the review, but this looks pretty good so far!

File: ADOPTERS.md (hunk 1 above)
Comment (brandon-powers) [reply ADOPTERS.md:31]: Ah yes, I'll clean that up & amend commit.

File: ADOPTERS.md (hunk 2 above)
Comment (brandon-powers) [reply ADOPTERS.md]: Good call, I've added the above suggestion & tweaked the wording a bit.

File: ADOPTERS.md (hunk 1 above)
Comment (brandon-powers) [reply ADOPTERS.md]: Fixed, `0ded6a1` reflects the single line addition.

**Comment by brandon-powers** [comment]
> Awesome @brandon-powers, thanks for taking the time to share your crossplane story with the community! a couple questions for you in the review, but this looks pretty good so far!

@jbw976 👍, glad to do so! Crossplane has certainly proved useful for us.

Fixed the auto-format & added a note on running in production, let me know what you think.

**Review by jbw976 — APPROVED** [review]
Amazing @brandon-powers, thank you for following up on the feedback and updating this PR - it looks perfect now!! 🙇‍♂️ 🤩

**Pull request merged**